src/Ogooreck.Sample.BusinessLogic.FSharp.Tests/Deciders/BankAccountTests.fs

```diff
@@ -0,0 +1,115 @@
+module BankAccountTests
+
+open System
+open Deciders.BankAccount
+open Deciders.BankAccountPrimitives
+open FsCheck
+open Ogooreck.BusinessLogic
+open Deciders.BankAccountDecider
+open FsCheck.Xunit
+
+let random = Random()
+let now = DateTimeOffset.UtcNow
```

Comment:
this is a static field which lives for as long as this test assembly is loaded.
I'd tend to use a type and put it as a field in the body that gets initialzied per test or in the body of the function

(if using fcheck and the specific value does not matter, let FsCheck gen it for you)

Reply:
Yeah, in those tests, I don't have any relation, it may be in fact any date. Changed to use FSCheck in https://github.com/oskardudycz/Ogooreck/pull/13/commits/9c3bd4fa7efd66b75bf111513ba31eea4892e0cb.